pages/linux/makepkg.md

```diff
@@ -19,3 +19,7 @@
 - Make a package, but skip source checksums:
 
 `makepkg --skipchecksums`
+
+- Clean the build folder before compiling:
```

Comment:
The `--cleanbuild` arg cleans the `src/` folder before build, the `--clean` arg removes the `src/` and `pkg/` folders after build.
```suggestion
- Clean up work directories after a successful build:
```